src/components/Explore/TokenTable/SearchBar.tsx

```diff
@@ -61,13 +63,15 @@ const SearchInput = styled.input<{ expanded: boolean }>`
 export default function SearchBar() {
   const [filterString, setFilterString] = useAtom(filterStringAtom)
   const [isExpanded, setExpanded] = useState(false)
+  const placeholderText = isMobile ? 'Search apps, tokens, and NFTs' : 'Search by name or token address'
```

Comment:
best to avoid this, can we just use one? if we need to do this it'd be better to set the content in css based on breakpoints

Reply:
let me ask design, they were different in the designs -- also I tried setting the placeholder content using CSS and i couldn't get it to work a while back, can try again though ahaha